index.md

```diff
@@ -1,34 +1,25 @@
-Anbox Cloud makes it possible to run Android apps in the cloud and supports all kinds of different use cases, including high-performance streaming of graphics to desktop and mobile client devices.
+Anbox Cloud enables running Android apps on any cloud platform at scale. It uses [LXD containers](https://linuxcontainers.org/) to run the nested Android containers and [Juju](https://charmhub.io/) for deployment in a cloud environment.
 
-Using container technology, Anbox Cloud manages the application life cycle while optimising for high density, performance and fast container boot times. It is scalable from a single-machine installation that can run scores of single Android systems to an entire cluster of machines. Anbox Cloud supports x86 and Arm64 hardware, providing the same set of features for both architectures. Deploying the solution is simple and straightforward, and it is based on powerful and battle-proven software from Canonical like [LXD](https://linuxcontainers.org/) and [Juju](https://jujucharms.com/).
+Anbox Cloud supports x86 and Arm64 hardware, providing the same set of features for both architectures. 
 
-Anbox Cloud uses system containers to emulate Android systems. This method results in an isolation and security level similar to virtual machines with an overhead as low as process containers. Therefore, Anbox Cloud can provide at least twice the container density compared to other Android emulation solutions, and can serve up to 100 Android instances per server.
+Since Anbox Cloud uses containerisation to emulate Android systems, you can achieve the isolation and security level of a virtual machine without the associated overhead. Therefore, compared to other Android emulation solutions, Anbox Cloud can provide at least twice the container density and can serve up to 100 Android instances per server.
```

Comment:
I think mentioning system containers is important here, or something similar (unprivileged containers?).
If we were using docker containers for instance, we would have very little isolation as we would need to run them with full privileges (hence a much weaker security and isolation).